README.adoc

```diff
@@ -0,0 +1,83 @@
+= Hazelcast Viridian Maven Plugin
+:description: The Hazelcast Viridian Maven Plugin is a Java development tool for testing and deploying link:https://docs.hazelcast.com/cloud/cluster-side-modules[cluster-side modules] on Serverless clusters.
+:page-plugin-version: 0.0.2
+
+image:https://maven-badges.herokuapp.com/maven-central/com.hazelcast.cloud/hazelcast-cloud-maven-plugin/badge.svg[link="https://maven-badges.herokuapp.com/maven-central/com.hazelcast.cloud/hazelcast-cloud-maven-plugin"]
+
+{description}
+
+The current version of the plugin is {page-plugin-version}.
+
+== Before you Begin
+
+Make sure that you have the following:
+
+- link:https://maven.apache.org/install.html[Maven]
+- The `JAVA_HOME` environment variable set to the path of JDK 8, 9, or 11
+
+== Usage Example
+
+[source,xml,subs="attributes+"]
+----
+<build>
+    <plugins>
+        <plugin>
+            <groupId>com.hazelcast.cloud</groupId>
+            <artifactId>hazelcast-cloud-maven-plugin</artifactId>
+            <version>{page-plugin-version}</version>
+            <configuration>
+                <apiBaseUrl>https://api.viridian.hazelcast.cloud</apiBaseUrl>
+                <clusterName>$\{clusterName}</clusterName>
+                <apiKey>$\{apiKey}</apiKey>
+                <apiSecret>$\{apiSecret}</apiSecret>
+            </configuration>
+        </plugin>
+    </plugins>
+</build>
+----
+
+== Configuration
+
+[cols="1m,2a,1m"]
+|===
+| Parameter|Description| Example
+
+| clusterName
+| Cluster name
+
+Find your cluster name in the link:https://cloud.hazelcast.com[Hazelcast Viridian console] by clicking *Connect Client*.
+| pr-1234
+
+| apiKey
+| API key
+
+Generate an API key and secret in the link:https://cloud.hazelcast.com/settings/developer[Hazelcast Viridian console].
```

Comment:
For the Viridian link should be https://viridian.hazelcast.cloud/settings/developer